bower.json

```diff
@@ -23,6 +23,6 @@
         "bower_components"
     ],
     "dependencies": {
-        "polymer": "Polymer/polymer#^0.5.1"
+        "polymer": "Polymer/polymer#^0.9.0"
```

Comment:
i'm not sure if 0.9.0 has been tagged yet. I think you might have to tag #0.9.0-rc.1 for bower to pick it up